### ISSUE TITLE
Can't mine

### ISSUE THREAD

**Issue by HunEQ** [UnamSanctam/SilentXMRMiner#456]
I tried mining with kawpow - Ravencoin and GPU only but it doesn't work. It connects to the pool and shows the worker, but soon after it stops so the pool reports 0.0 MH/s. Waited 30 minutes but it did not change.
Tried two different pools and different ports but still doesn't work.

**Comment by UnamSanctam** [comment]
What GPU do you have? Use something like MSI Afterburner or something similar to see if it's mining.

**Comment by HunEQ** [comment]
I'm on a 2070 Super Mobile. Maybe it doesn't work for laptops? I'm just testing for now.

When I check temps it goes up at the start but then it cools down and no other miner behaves like this.
30 minutes and no income reported in the pool, I don't think it's mining and I'm not quite sure what I'm doing wrong. XMR mining works fine.

Unrelated, but can you run two  instances and mine XMR and RVN on the same computer?

**Comment by UnamSanctam** [comment]
It should work for laptops, I've tried it on mine and that worked. You can run two at once but you should then choose different "Inject Into" for both and the two watchdogs (if enabled on both) might "fight" each other but that doesn't matter much. Have you ever mined Ravencoin with any other miner and it worked? Personally I can't mine Ravencoin at all no matter which miner I use (tried all miners that supports Ravencoin) which is why I'm wondering in case the problem is similar. It going up at the start is normal for Ravencoin (and others like Ethereum) since it's generating the DAG and everything else, can you run this [MinerCheck3.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/7278702/MinerCheck3.zip) and screenshot what it reports (but censor your wallet address)?

**Comment by HunEQ** [comment]
Yes, I've checked and T-rex works for mining Ravencoin on this laptop.

Here you go

![MinerCheck](https://user-images.githubusercontent.com/91876221/135877375-8e53ae23-5869-4e66-9e8d-73c37323dfc0.png)

**Comment by UnamSanctam** [comment]
Can you try mining with https://github.com/xmrig/xmrig and see what happens (since that is what the miner is based on)? You can copy the miner arguments in your screenshot there and run the xmrig with those in case it has something to do with xmrig.

**Comment by HunEQ** [comment]
It won't let me use CUDA, don't know why. It tries to mine using CPU even if disabled and KawPow is not compatible. 
Does it look like an issue on xmrig's side?

**Comment by UnamSanctam** [comment]
Sorry for the late response, are your drivers updated?

**Comment by HunEQ** [comment]
I've reinstalled the drivers and CUDA library and it finally works now. Thank you very much!

**Comment by UnamSanctam** [comment]
Ah great that it works, good job.
